### PR TITLE
V0.1 TFT Mount added

### PR DESCRIPTION
**Pull request by Eifel-Joe**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
This is a mod for installing a Waveshare 4.3inch TFT in the front skirt of the V0.1. Therefore the feet has to be enlarged by 10mm an the font skirt has to be modified. Only optical parts were modified.

#### How was it tested? 
Part were printed and mounted. They fitted on the frame.

#### Any background context you want to provide?
No.

#### Screenshots (if appropriate)
Screenshot of the CAD (rendered).

#### Further notes
CAD for the mod is included (Fusion360)

### CODE REVIEW

**Comment by Eifel-Joe** [comment]
IMG_8367 (2)

Picture of the mounted mod.

File: printer_mods/Eifel-Joe/V0.1_Waveshare_4.3_inch_TFT_Mount/Readme.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Voron v0.1 4.3" Touchscreen Mount
+
+This mod mounts a [Waveshare 4.3inch Capacitive Touch Display](https://www.waveshare.com/4.3inch-dsi-lcd.htm) purchased from [Amazon](https://www.amazon.de/gp/product/B08CH24YYD/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1).  The screen is oriented at a 55 degree angle
+
+<div align="center">
+    <img src="Image/Waveshare43_Mount.png" height="400">
+</div>
```
Comment (MapleLeafMakers) [inline printer_mods/Eifel-Joe/V0.1_Waveshare_4.3_inch_TFT_Mount/Readme.md:7]: No HTML in Markdown flies please.

Comment (Eifel-Joe) [reply printer_mods/Eifel-Joe/V0.1_Waveshare_4.3_inch_TFT_Mount/Readme.md]: -Spaces in filenames are removed
-STEP format for CAD added
-README.md new. Don't know what happend last time
-Photo above added to README
-Removed HTML-Code

**Comment by Eifel-Joe** [comment]
-Spaces in filenames are removed
-STEP format for CAD added
-README.md new. Don't know what happend last time
-Photo above added to README

**Comment by MapleLeafMakers** [comment]
These filenames still contain spaces:

```printer_mods/Eifel-Joe/V0.1_Waveshare_4.3_inch_TFT_Mount/STL/Mount_Waveshare 4.3 DSI v2_Mount_Left.stl
printer_mods/Eifel-Joe/V0.1_Waveshare_4.3_inch_TFT_Mount/STL/Mount_Waveshare 4.3 DSI v2_Mount_Right.stl
printer_mods/Eifel-Joe/V0.1_Waveshare_4.3_inch_TFT_Mount/CAD/Waveshare 4.3.f3d
printer_mods/Eifel-Joe/V0.1_Waveshare_4.3_inch_TFT_Mount/CAD/Waveshare 4.3.step```

**Comment by Eifel-Joe** [comment]
I'm sorry. Didn't notice these spaces. Filenames should now be fine.

**Review by cbc02009 — CHANGES_REQUESTED** [review]

File: printer_mods/Eifel-Joe/V0.1_Waveshare_4.3_inch_TFT_Mount/Readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Voron v0.1 4.3" Touchscreen Mount
+
+This mod mounts a [Waveshare 4.3inch Capacitive Touch Display](https://www.waveshare.com/4.3inch-dsi-lcd.htm) purchased from [Amazon](https://www.amazon.de/gp/product/B08CH24YYD/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1).  The screen is oriented at a 55 degree angle
```
[the remark this answers is not in the document]
Comment (Eifel-Joe) [reply printer_mods/Eifel-Joe/V0.1_Waveshare_4.3_inch_TFT_Mount/Readme.md]: Tracking Data is removed. I Didn't know that, thought it was part of the link, but i will remember this.

File: printer_mods/README.md [hunk 3]
```diff
@@ -145,6 +145,7 @@ like so:
 | |[Short Z Joints](./Ellis/Short_Z_Joints) | Shorter Z joints for the v2.4 |:x: |:x: |:heavy_check_mark: |:x:|
 | |[Single MGN9H Carriage](./Ellis/Single_MGN9H_Carriage) | Single MGN9H toolhead carriage with toolhead x endstop support (for umbilical) |:x: |:heavy_check_mark: |:heavy_check_mark: |:grey_question:|
 | ElPoPo | [Removable Doors](./ElPoPo/RemovableDoors) | New hinge to easily remove the door and have a wide opening  V2.x | :x: | :x: | :heavy_check_mark: | :x:|
+| Eifel-Joe | [V0.1 Waveshare 4.3 inch TFT Mount](./Eifel-Joe/V0.1_Waveshare_4.3_inch_TFT_Mount) | V0.1 4.3 inch TFT Mount | :heavy_check_mark: | :x: | :x: | :x: | :x: |
```
[the remark this answers is not in the document]
Comment (Eifel-Joe) [reply printer_mods/README.md]: You are right. I changed it to go after edwardyeek's.

**Review by cbc02009 — APPROVED** [review]

**Comment by FHeilmann** [comment]
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro

**Pull request closed without merging**